docs/source/building-applications.rst

```diff
@@ -4,6 +4,12 @@
 Building Applications
 =====================
 
+The Building Applications section covers the elements that are used to create, extend, and test your Daml full-stack application (including APIs and script libraries) and the architectural best practices for bringing those elements together.  
+
+As with the Writing Daml section, you can find the Daml code for the example application and features `here <https://github.com/digital-asset/daml/tree/main/docs/source/daml/intro/daml>`_ or download it using the Daml assistant. For example, to load the sources for section 1 into a folder called 1_Token, run daml new 1_Token --template daml-intro-1. 
```

Comment:
You know better then me of course, but intuitively: should "Writing Daml" be in quotes?

Reply:
I think in this context using title case is sufficient, and matches how we refer to other documents or sections of documents elsewhere in our docs.